content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md

```diff
@@ -32,35 +32,23 @@ When enabling non-interactive access to RKVST, you ***must*** create your first
 
 ### Using the RKVST UI to create an App Registration (First-Time Setup)
 
-1. As an Administrator, open the `Settings` interface.
```

Comment:
This block is mostly indentation formatting to tighten up the whitespace and make the imagery indented under the numbered lists.